exocom-server/bin/features

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env bash
+set -ex
 
 node_modules/o-tools-livescript/bin/build
-if [ "$#" == "0" ]; then
-  node_modules/.bin/cucumber-js --tags ~@todo
+if [ "$#" != "1" ]; then
```

Comment:
The original intent was that when running `features`, it prints no output. When running `features foo.feature`, it prints debug output. I'd like to keep this behavior, since it is helpful for debugging. If we need something else on CI, let's modify `spec_ci`, it defines the exact commands to run on CI.